apps/molgenis-components/src/components/forms/BaseInput.vue

```diff
@@ -16,6 +16,10 @@ export default {
       type: String,
       required: false
     },
+    placeholder: {
+      type: String,
+      required: false
+    },
     helpText: {
```

Comment:
what is this? If this is the 'description' in emx2 terminology then please please call this 'description'. I want this to be a tightly coupled framework to emx2 terminology where possible to make it easier to understand as a developer.

Reply:
placeholder is a html attr, name a html for name, a label is a html label , the id is a html id.